Added motivation (#27)
opengeospatial/geosparql3dwhitepaper: document/sections/03a-motivation.adoc

 If one desires digital, automated processing or analysis of 3D spatial questions, spatial information must be structured in an explicitly machine-readable format. When this is done, initiatives based on federated systems with heterogeneous datasets can obtain computer-generated answers to spatial 3D queries without requiring conversion. This highlights the importance of semantic alignment and standardized geometry processing in machine-driven analysis. <<Göbel2024>>

 When performing federated queries on heterogeneous spatial datasets without conversion, a common vocabulary is required to semantically define the 3D object and its 3D spatial function. For both humans and machines, it must be unambiguous what is meant by a query such as:

>  “Does the '3D geometry' of the proposed construction '3D-intersect' with the '3D geometry'of an existing asset owned by another party?”

Reviewer: ```suggestion
> “Does the '3D geometry' of the proposed construction '3D-intersect' with the '3D geometry'of an existing asset owned by another party?”
```

>  It should not matter whether the data is provided in a 3D-GIS, 3D-BIM or 3D-CG standard vocabulary.  

Reviewer: This sentence is orphaned here. Suggest merging with the paragraph above.

=== Added value of 3D in the GeoSPARQL Vocabulary for Stakeholders

> ==== We aim to support data Consumers by:

Reviewer: Not sure "We" is the option here. I would start this sentence like: "A geo-spatial 3D specification would support data Consumers by"

> ==== We aim to support data Consumers by:

Reviewer: ```suggestion
==== We aim to support data Consumers by providing:
```

 - access and understanding of 3D-geometry and 3D-topology from multiple sources; 
 - the integration of 3D-data geospatial and non-geospatial;
 - data quality evaluation of the given 3D-data;

>  - Geometric and topological description in natural language.

Reviewer 2: I don't completely understand this point, "geometric and topological description in _natural language_". How would adding 3D capability to GeoSPARQL help provide descriptions in natural language? I thought the point is to make it machine readable.

==== We aim to support Data Providers by:

>  - Maximize the use of 3D-geometry and 3D-topology data;

Reviewer: ```suggestion
 - Maximizing the use of 3D-geometry and 3D-topology data;
```

 - Defining multiple 3D topology or geometries;

>  - To quality-assure 3D geospatial data.

Reviewer: ```suggestion
 - assuring the quality of 3D geospatial data.
```

=== Semantic-functional value of 3D-GeoSPARQL
 Value for data reasoning, formal description, and semantic interoperability

==== Meaningful creation and querying of 3D geometry

>  Whether manually or programmatically defining 3D geometries in BIM, GEO or CG environments, the GeoSPARQL standard could provide or connect to vocabulary to represent 3D structures. A 3D cube, for example, can be represented in multiple ways:

Reviewer: ```suggestion
Building on 3D geometries defined in BIM, GEO or CG, a GeoSPARQL 3D specification would provide precise semantic associations to a vocabulary of 3D structures. A 3D cube, for example, can be represented in multiple ways:
```

Reviewer 2: What does CG stand for? I didn't see the abbreviation explained in earlier text. (could be I'm missing some context)

 - As a set of bounding points, lines, and faces;
 - As a base surface with an extrude function;

>  - As an 3D geometric primitive (e.g., the concept of a 'cube' as a spatial figure).

Reviewer: ```suggestion
 - As a 3D geometric primitive (e.g., the concept of a 'cube' as a spatial figure).
```

>  When GeoSPARQL offers the means to define and link 3D geometries in these different ways, it allows both humans and machines to interpret and interact with the data effectively.

Reviewer: ```suggestion
 Once GeoSPARQL offers the means to define and link 3D geometries in these different ways, it will allow both humans and machines to interpret and interact with the data effectively.
```

==== Meaningful creation and querying of 3D topological relationships
 Is “3D Geometry A” inside, touching, overlapping, intersecting or above “3D Geometry B”?

 Being able to describe and use such topological relationships is crucial for conducting spatial analyses, formulating rules, and deriving knowledge from differten heterogeneous datasets.

 Beyond describing topological relationships, one should be able to query them:

>  What is the spatial relationship between 3D Geometry A and 3D Geometry B?

Reviewer: ```suggestion
> What is the spatial relationship between 3D Geometry A and 3D Geometry B?
```

 Such queries should return the appropriate topological result (e.g., "intersects"), which is essential for use cases like clash detection in design validation

==== Derivation of geometric and topological knowledge
 From explicitly modeled geometric and topological data, one should be able to infer implicit knowledge.

>  For instance, if 3D Geometry A is 3D inside 3D Geometry B, then we can infer that:

Reviewer: ```suggestion
 For instance, if 3D Geometry A is inside 3D Geometry B, then we can infer that:
```

 A does not "touch" B, and
 A does not "overlap" with B,

 even if these facts are not explicitly stated.

 For entailment we aim to be able to answer queries like: "Does this tree have a representation in 3D geometry?"

 If the tree is defined using a MultiSolid geometry, then the answer should be "yes," even if this is not explicitly declared as such.

> ==== Adding multiple 2D and 3D geometries to a single feature

Reviewer: In this section it is important to note that by GeoSPARQL already supports this multiple geometric definition of a feature.

 GeoSPARQL supports linking multiple geometries (e.g., 2D and 2.5D) to a single feature. We aim to extend this with 3D geometries. 

 For example, a tree in the real world may be represented in databases with both a 2D and a 3D geometry.

 This should not require duplication of the tree entity — there can be one tree instance with both geometries linked, avoiding redundancy in the data model.

> === Value for user experience, visualization, and application use of 3D-GeoSPARQL

Reviewer 2: I can imagine everything under this heading being moved to the Benefits section. Perhaps replaced by one or two sentences in the Motivation giving some examples of expected benefits, because these benefits have motivated us of course.
